instrumentor-core/src/main/java/com/sproutsocial/metrics/Instrumentor.java

```diff
@@ -31,11 +26,15 @@
 
 
     private class Context {
+        private Meter totalMeter;
+        private Meter successMeter;
         private Meter errorMeter;
         private Timer timer;
         private Counter inFlight;
 
         Context(String name) {
+            totalMeter = metricRegistry.meter(name + ".total");
```

Comment:
Thanks! Somehow I missed that Timer is also a meter and can report the total number of calls. I have removed the total meter. 
I do believe we should keep the success rate instead of implying success rate being the difference of total and error in order to make the success meter directly comparable to the error meter.